okhttp-tests/src/test/java/okhttp3/CallTest.java

```diff
@@ -3560,6 +3563,8 @@ private RequestBody requestBody(final boolean chunked, final long size, final in
 
     assertTrue(source.exhausted());
     assertEquals(Headers.of("trailers", "boom"), response.trailers());
+
+    response.close();
```

Comment:
Should this be try with resources? Do you want everything closing cleanly even when tests fail?

Reply:
Fixed.